edu_lit_app/lit_app/views.py

```diff
@@ -1,3 +1,13 @@
+from django.http import HttpResponse
 from django.shortcuts import render
+from lit_app.models import * 
+
+def todo(request):
+    data = {
+        "tests": Test.objects.all()
```

Comment:
Would we want to modify this to only include tests that a student hasn't taken before?